app/models/report.rb

```diff
@@ -62,7 +62,7 @@ def self.inherited(child)
   # extracts serialized metrics and keep them as a hash_with_indifferent_access
   def metrics
     return {} if self[:metrics].nil?
-    YAML.safe_load(read_metrics).with_indifferent_access
+    YAML.safe_load(read_metrics, permitted_classes: [ActiveSupport::HashWithIndifferentAccess]).with_indifferent_access
```

Comment:
The alternative solution would be to make `read_metrics` not return HWIA, but I wasn't sure the mangling in that function is the only way for the data to get HWIA header, so this seemed safer.